payments-core/src/main/java/com/stripe/android/payments/StripeBrowserLauncherActivity.kt

```diff
@@ -33,7 +33,12 @@ internal class StripeBrowserLauncherActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val args = PaymentBrowserAuthContract.parseArgs(intent)
+        val args = try {
+            PaymentBrowserAuthContract.parseArgs(intent)
+        } catch (e: Exception) {
+            null
```

Comment:
It's helpful to log some error here.

Reply:
Any recommendations on an easy way to do that?

Reply:
Just use the Android logger, `Log.e(TAG, "Error parsing arguments", e)`.

Reply:
I would like you to reconsider.  This is a scenario that should never happen, and having a place where we aren't using the same logger as everywhere else is not a good practice.  Getting our shared logger into this class is not really worth the extra lift.

Reply:
I think failing silently without any logs is worse than using the default Android logger, but it's up to you

Reply:
isn't our logger a singleton now? Can't you just do `Logger.getInstance(enableLogging).error("oops")`?